src/u-blox_structs.h

```diff
@@ -854,6 +854,57 @@ typedef struct
   UBX_NAV_CLOCK_data_t  *callbackData;
 } UBX_NAV_CLOCK_t;
 
+// UBX-NAV-TIMELS (0x01 0x26): Leap second event information
+const uint16_t UBX_NAV_TIMELS_LEN = 24;
+
+typedef struct
+{
+  uint32_t iTOW; // GPS time of week of the navigation epoch: ms
+  uint8_t version; // Message version (0x00 for this version)
+  uint8_t reserved1[3];
+  uint8_t srcOfCurrLs; //Information source for the current number of leap seconds
+  int8_t currLs; //Current number of leap seconds since start of GPS (Jan 6, 1980), s
+  uint8_t srcOfLsChange; //Information source for the future leap second event
+  int8_t lsChange; //Future leap second change if one is scheduled, +1, 0, -1s
+  int32_t timeToLsEvent; //Num of secs until the next or from the last leap second, s
+  uint16_t dateOfLsGpsWn; //GPS week num (WN) of the next or the last leap second event
+  uint16_t dateOfLsGpsDn; //GPS day of week num (DN) for the next or last leap second event
+  uint8_t reserved2[3];
+  int8_t valid; // Validity flag, 1 = valid, otherwise 0
```

Comment:
Hi @UT2UH ,
Looking at the ZED_F9P interface description, ```valid``` contains two bits (not one):

![image](https://user-images.githubusercontent.com/5690545/114838088-43b8b600-9dcc-11eb-9bd0-db7e87c8e8e1.png)

(Sometimes the ZED messages contain extra bits compared to the M8 family)
Can you please change ```valid``` into a union+struct? Something like:

```
  union
  {
    uint8_t all;
    struct
    {
      uint8_t validCurrLs : 1; // 1 = Valid current number of leap seconds value
      uint8_t validTimeToLsEvent : 1; // 1 = Valid time to next leap second event or from the last leap second event if no future event scheduled
    } bits;
  } valid;
```

This means you will need to change ```getLeapIndicator``` so it uses ```data.valid.bits.validCurrLs```.
Thanks!
Paul